Makefile

```diff
@@ -288,6 +297,26 @@ $(LIBUV):
 	@echo Building libuv...
 	$(SHOW)$(MAKE) --no-print-directory -C build/libuv DEBUG=''
 
+ifeq ($(DEBUG),1)
+RUST_BUILD_MODE=
+else
+RUST_BUILD_MODE=--release
+endif
+ifeq ($(DEBUG),1)
+RUST_ARTIFACT_SUBDIR=debug
+else
+RUST_ARTIFACT_SUBDIR=release
+endif
+
+redisearch_rs:
+	@echo Building redisearch_rs..
+	$(SHOW)cd $(REDISEARCH_RS_DIR) && cargo build $(RUST_BUILD_MODE)
+	$(SHOW)mkdir -p $(REDISEARCH_RS_BINDIR)
+	$(SHOW)cp -r $(REDISEARCH_RS_DIR)/target/$(RUST_ARTIFACT_SUBDIR)/*.a $(REDISEARCH_RS_BINDIR)
```

Comment:
Maybe we can configure rust to build the artifacts directly to `REDISEARCH_RS_BINDIR`

Reply:
Done in [1068b0f](https://github.com/RediSearch/RediSearch/pull/5694/commits/1068b0f504dfa96408759c8e3ef239f77a5e85a4)